lib/plugins.ts

```diff
@@ -65,7 +65,7 @@ export interface ProtractorPlugin {
    *     for the promise to resolve before continuing.  If the promise is
    *     rejected, a failed assertion is added to the test results.
    */
-  teardown?: () => q.Promise<any>;
+  teardown?(): void|q.Promise<void>;
```

Comment:
These can actually return any kind of promise.  How about `void|q.IPromise`?  Also, update the comment to have a return type `{Promise=}`.